src/ng/directive/input.js

```diff
@@ -1593,19 +1589,16 @@ function rangeInputType(scope, element, attr, ctrl, $sniffer, $browser) {
     setInitialValueAndObserver(maxAttrType, 'max', maxChange);
   }
 
-  function setInitialValueAndObserver(attrType, attrName, changeFn) {
-    var initialAttrValue;
-    if (attrType === attrName) {
-      initialAttrValue = attr[attrName];
-      // Set the actual element attribute so that the browser can adjust the value based on
-      // the max value, which might be interpolated
-      element.attr(attrName, initialAttrValue);
+  function setInitialValueAndObserver(actualAttrName, htmlAttrName, changeFn) {
+    // e.g. max === max
+    if (actualAttrName === htmlAttrName) {
+      // interpolated attributes set the attribute value only after a digest, but we need the
+      // attribute value when the input is first rendered, so that the browser can adjust the
+      // input value based on the min/max value
+      element.attr(htmlAttrName, attr[htmlAttrName]);
```

Comment:
So, if I understand correctly, when using `min`/`max` (with or without interpolation) the browser will set an initial value (if none is set) to `(max + min) / 2`. But if you use `ngMin`/`Max` the browser will ignore them and still assume the default `min`/`max` values (0 and 100 resp.) and set the initial value of the input to 50.
Is that correct?

Additionally, when using `ngMin`/`Max`, the browser will ignore their values and only allow values between the default 0 and 100.
This doesn't sound very useful... :confused: